config/config.go

```diff
@@ -46,6 +47,7 @@ func Get() (*Config, error) {
 		ZebedeeURL:                 "http://localhost:8082",
 		HealthCheckInterval:        30 * time.Second,
 		HealthCheckCriticalTimeout: 90 * time.Second,
+		KafkaVersion:               "1.0.2",
```

Comment:
If this is being set here you can use `cfg.KafkaVersion` in main when you need to reference it. Being slightly picky I know but having variables defined in a single place makes updating/changing things easier down the line.